### PR TITLE
onboarding syncing updates

### LINKED ISSUE #15787: Design notes: sign in by syncing
## Problem 
Suggestions/issues on implementation in Figma are [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?node-id=1929-154137&t=IygHPkoLwgiVR8Xa-4)

### Empty state

- [x] Update icon (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429151296))
- [x] Update tabs color (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429121879))
- [x] Update tabs color (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429121879))
- [x] Update dash line border (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429121577))
- [x] Update icon "Enable camera" (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429120707))
- [x] Update color "Enable camera" (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429121021))
- [x] Update background  (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429120328))
- [x] Update font size  (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429147530))

### Camera access
- [x] Update back nav(full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429210625))
- [x] Check borders design(full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429224561))
- [ ] Add flash option (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429186829))

### Syncing devices
- [x] Remove navigation (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429189482))
- [ ] Add illustration  (full comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=1929:154137#429189321))

### PR DESCRIPTION
**Pull request by jo-mut**
fixes #15787

### Summary
Resolves the issues pointed out except illustrations and support camera flash functionality. Seems the camerakit library has some issues with flash as it does not seem to work on both android and ios

Screenshot 2023-06-09 at 17 44 53
Screenshot 2023-06-09 at 17 45 02

### CODE REVIEW

**Comment by ulisesmac** [comment]
Hi @jo-mut 

I was trying to review and contrast the changes, but it's being a little hard to know what exactly expect. 
Could you please add some screenshots of the changed/implemented views and list what is being fixed?
Thank you in advance!

btw, this PR has the "fixes #..." but there're two remaining tasks in the issue, so I'd suggest to open new issues for those tasks or not closing this issue with this PR. wdyt?

Thanks again.

**Comment by jo-mut** [comment]
@ulisesmac I have added some screenshots to the p.r. As you might  have seen some of the issues are not addressed in this pr. They can be addressed in a separate issue if the Q.A agrees with that and once the Q.A is okay with this pr it would then be fine to open a new issue.

There seems to be an issue with the camerakit as some of the camera features dont seems to work on both android and ios which is why they will better addressed in a separate issue as they have to do with the camerakit library we are using

**Comment by ulisesmac** [comment]
Thank you!

File: src/status_im2/contexts/onboarding/sign_in/style.cljs [hunk 1]
```diff
@@ -62,6 +62,8 @@
    :background-color  colors/white-opa-5
    :border-color      colors/white-opa-10
    :border-radius     12
+   :border-style      :dashed
+   :border-width      1
```
Comment (ulisesmac) [inline src/status_im2/contexts/onboarding/sign_in/style.cljs:66]: How this change is showed?
I think this  `def` (`camera-permission-container`) is not being used :thinking:  inspecting a little more the code in this file, it seems like many `def`s are not being used, probably it's unrelated, but would you mind removing these unused vars if you are going to touch this file?

thanks!! :)

File: src/status_im2/contexts/syncing/scan_sync_code/view.cljs [hunk 2]
```diff
@@ -142,21 +143,63 @@
...
      [rn/view {:style (style/viewfinder-container qr-view-finder)}
-
-      [rn/view {:width size :height size :justify-content :space-between}
-       [rn/view {:flex-direction :row :justify-content :space-between}
-        [border :border-top-width :border-left-width :border-top-left-radius]
-        [border :border-top-width :border-right-width :border-top-right-radius]]
+      [rn/view
+       {:style (style/qr-view-finder-container size)}
+       [rn/view
+        {:flex-direction :row :justify-content :space-between}
```
Comment (ulisesmac) [inline src/status_im2/contexts/syncing/scan_sync_code/view.cljs]: Please add these styles inside a `:style` key and move them to the style namespace

File: src/status_im2/contexts/syncing/scan_sync_code/view.cljs [hunk 3]
```diff
@@ -142,21 +143,63 @@
...
+           :top   -1}]
+         [border-tip
+          {:left   -1
+           :bottom 0}]]]
+       [quo/button
+        {:icon                true
+         :type                :blur-bg
+         :size                32
+         :accessibility-label :camera-flash
+         :on-press            #()
```
Comment (ulisesmac) [inline src/status_im2/contexts/syncing/scan_sync_code/view.cljs]: If we are not going to use the `on-press` call back we could remove the key from the map, I think that's better than providing an empty function

**Review by ulisesmac — APPROVED** [review]
Thanks for the PR! The screen  has been improved! :100:

File: src/status_im2/contexts/onboarding/sign_in/style.cljs (hunk 1 above)
Comment (smohamedjavid) [reply src/status_im2/contexts/onboarding/sign_in/style.cljs:66]: I guess I knew the answer to it. 

A refactor happened last month (https://github.com/status-im/status-mobile/pull/15623) where the sign-in code has been moved to a common component (`scan-sync-code`) for making it easy to use from "Sign In" and "Add Device/Scan Sync Code" screens. But, the styles of sign-in haven't been removed. We can delete it.

File: src/status_im2/contexts/syncing/scan_sync_code/view.cljs [hunk 4]
```diff
@@ -142,21 +143,63 @@
 (defn- border
   [border1 border2 corner]
   [rn/view
-   (assoc {:border-color colors/white :width 80 :height 80} border1 2 border2 2 corner 16)])
+   (assoc style/border border1 2 border2 2 corner 16)])
```
Comment (smohamedjavid) [inline src/status_im2/contexts/syncing/scan_sync_code/view.cljs]: Kindly move these styles as value to `:style` key.

**Review by smohamedjavid — APPROVED** [review]
Nice work! 🚀

File: src/status_im2/contexts/syncing/scan_sync_code/view.cljs [hunk 5]
```diff
@@ -142,21 +143,63 @@
...
+         [border :border-top-width :border-left-width :border-top-left-radius]
+         [border-tip
+          {:right -1
+           :top   0}]
+         [border-tip
+          {:left   0
+           :bottom -1}]]
+        [rn/view
+         [border :border-top-width :border-right-width :border-top-right-radius]
+         [border-tip
```
Comment (ilmotta) [inline src/status_im2/contexts/syncing/scan_sync_code/view.cljs]: We can join the lines into a single one, as in:

```clojure
[rn/view
 [border :border-top-width :border-left-width :border-top-left-radius]
 [border-tip {:right -1 :top 0}]
 [border-tip {:left 0 :bottom -1}]]
```

**Review by ilmotta — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
Hi @jo-mut .Thank you for PR. Here are some found issues:

## ISSUE 1: Icon is darker than intended in the design
### Actual result:
image

### Expected result:
image

### Device:
- iPhone 11 Pro, IOS 16
- Pixel 7a, Android 13

**Comment by VolodLytvynenko** [comment]
## ISSUE 2: [Android] Horizontal Frames are is still not on middle of camera box and a bulge is shown at the end of the frame line

### Actual result:
- The horizontal frames are not on the middle of camera box
- Some bulge is shown at the end of the frame lines
image

### Expected result:
image

### Device:
- Pixel 7a, Android 13

**Review by ibrkhalil — APPROVED** [review]

**Comment by jo-mut** [comment]
@VladimrLitvinenko thank your observations. These issues are now resolved.

**Comment by jo-mut** [comment]
Screenshot 2023-06-14 at 21 01 21

ios look

**Comment by jo-mut** [comment]
how it looks on android

**Comment by VolodLytvynenko** [comment]
> 
> 
> how it looks on android

HI @jo-mut. Issue 1 is fixed. Issue 2 is reproduced for some devices. Reported it as a follow-up https://github.com/status-im/status-mobile/issues/16278 with low priority. Current PR can be merged. Thanx

**Comment by jo-mut** [comment]
@VladimrLitvinenko the issue addressed in this pr has three bugs that are not yet resolved

- add flash option 
  This can be addressed in a follow-up pr as it needs this https://github.com/status-im/status-mobile/pull/16248 to be merged first. The underlying issue is the camarekit library. The issue will be tracked with this https://github.com/status-im/status-mobile/issues/16243

**Comment by VolodLytvynenko** [comment]
> * This can be addressed in a follow-up pr as it needs this

Hi, @jo-mut. Just to confirm, did I understand correctly that the current PR is waiting for the completion of https://github.com/status-im/status-mobile/pull/16248  before it can be retested and merged?

**Comment by jo-mut** [comment]
@VladimrLitvinenko no the current pr is fully done. I mean to say the remaining add flash to camera will be addressed in another pr. Just stating for the record but this is okay to merge

**Comment by VolodLytvynenko** [comment]
> no the current pr is fully done. I mean to say the remaining add flash to camera will be addressed in another pr. Just stating for the record but this is okay to merge

Got it. Thank you for the clarification

**Pull request merged**